src/app/globals.css

```diff
@@ -25,3 +25,28 @@ body {
     )
     rgb(var(--background-start-rgb));
 }
+
+h1 {
```

Comment:
[Please Change]: Let's use CSS modules to separate the styling here; as in #12 (I know we had simplified it at the time, but they figured out the heavy lifting and it's best to avoid any potential style conflicts after both these PRs are merged).

Reply:
Done